handbook/engineering/hiring/engineering-manager-code-intelligence.md

```diff
@@ -0,0 +1,70 @@
+# Engineering Manager - Code Intelligence
+
+We are looking for an engineering manager to lead our team that is building the semantic graph of code. The team's mission is to build a code intelligence platform that understands code in all languages and provides rich metadata to power the rest of Sourcegraph, including code search, insights, and campaigns. Read the [code intelligence team handbook documentation](https://about.sourcegraph.com/handbook/engineering/code-intelligence) to learn more.
+
+## About you
+
+- You know how to authentically inspire, challenge, and grow individuals and teams.
+- You are motivated to work on ambitious long term goals to solve complex technical challenges.
+- You are skilled at breaking up large problems (high complexity, high risk), into smaller achievable milestones (lower complexity, lower risk) that the team can iterate on.
+- You have experience working with multiple programming languages and their ecosystems (e.g., tooling, communities), and are interested and curious to engage with languages that you aren't yet familiar with.
+- You are [high agency](https://twitter.com/shreyas/status/1276956836856393728).
+
+## Your responsibilities
+
+- You will facilitate a cohesive team environment where everyone is happy, motivated, and productive.
+- You will ensure that the team has [well defined goals that are aligned with the business](https://about.sourcegraph.com/handbook/engineering/code-intelligence#goals), and that the team is making continuous iterative progress towards those goals every week.
+- You will help engineers grow in their careers.
+- You will hire exceptional engineers to grow the team (from 3 to ~6-8 engineers) in a sustainable way so that the team can accomplish more over time.
+
+## Qualifications
+
+- You have excellent written and verbal communication.
```

Comment:
How is excellence in this skill set demonstrated for us? Try to not use subjective terms that can have a wide range of interpretation by the reader. 

Okay with deleting this line as well, since this is a type of skill that we would usually expect from all candidates. 

Suggestion if keeping: 
- You are able to effectively communicate (both written and verbal) goals, timelines, and product direction to all levels of the company.

Reply:
> How is excellence in this skill set demonstrated for us?

Looking at the candidate's writing in their resume, application question responses, LinkedIn profile, etc. I view an application negatively if I see misspelled words, incomplete sentences, or, more importantly, not communicating the right things (did they organize their resume to communicate their accomplishments effective, did they answer the application questions effectively). If you think this is baseline and not worth mentioning, then we can remove from qualifications (but I might add something above in the "about me" to make it clear that we care about this a lot).

Reply:
I made some updates, including removing this explicit line. Can you take another look?